server/tserver/src/test/java/org/apache/accumulo/tserver/log/SortedLogRecoveryTest.java

```diff
@@ -71,6 +70,10 @@
   static final Text cq = new Text("cq");
   static final Value value = new Value("value");
 
+  @Rule
+  public TemporaryFolder tempFolder =
+      new TemporaryFolder(new File(System.getProperty("user.dir") + "/target"));
```

Comment:
This is cool.  Won't this delete the entire target directory though? If that is the case then I would imagine this would interfere with the build. Or is it anything that is created under this test that gets removed?  It is not clear from the javadoc.

Reply:
The argument to the constructor is the parent folder in which the temporary directory is created. This is from the javadoc for TemporaryFolder. When the test completes, the `@Rule` makes it delete itself, but not the parent directory.

Reply:
Thanks.  I missed that the parameter is parentFolder.